lib/DBDefs/Default.pm

```diff
@@ -136,6 +136,9 @@ sub CANONICAL_SERVER          { "https://musicbrainz.org" }
 # The server used to link to CritiqueBrainz users and reviews.
 sub CRITIQUEBRAINZ_SERVER     { "https://critiquebrainz.org" }
 
+# The URL where static resources are located, excluding the trailing slash.
+sub STATIC_RESOURCES_LOCATION { 'http://localhost:5000/static' }
```

Comment:
Even just mentioning in INSTALL.md that the setting exists and needs to be customized if you start plackup on any port other than 5000 would have helped.